cli/cmd/compliance.go

```diff
@@ -435,3 +441,49 @@ func statusToProperTypes(status string) string {
 		return "Unknown"
 	}
 }
+
+func validateRecommendationID(s string) bool {
```

Comment:
nit: since all places where you use this function is inside an `if` statement, it looks better to me this way:

```suggestion
func validRecommendationID(s string) bool {
```

Reply:
We can add more inputs of valid ID's to the tests